packages/docs/core-concepts/plugins.md

```diff
@@ -365,18 +365,23 @@ When [using pinia alongside Nuxt](../ssr/nuxt.md), you will have to create a [Nu
 import { PiniaPluginContext } from 'pinia'
 import { Plugin } from '@nuxt/types'
 
-function MyPiniaPlugin({ store }: PiniaPluginContext) {
+const MyPiniaPlugin = ({ store }: PiniaPluginContext) => {
   store.$subscribe((mutation) => {
     // react to store changes
     console.log(`[🍍 ${mutation.storeId}]: ${mutation.type}.`)
   })
 
-  return { creationTime: new Date() }
+  const creationTime = new Date();
+   
+  // add new state
+  store.$state.creationTime = creationTime;
+  store.secret.creationTime = creationTime;
```

Comment:
Why are you adding this to state and secret? Returning it as properties was good enough for the purpose of the example. I think we should keep the previous version.

Reply:
In order to make the typing work, you need to add the TS definitions as shown in the Plugin section

Reply:
I got confused because at the top of the file it says `// plugins/myPiniaPlugin.js' but the content is Typescript and it's also stated below `Note the above example is using TypeScript...`